README.md

```diff
@@ -97,7 +97,12 @@ Local-Android | ```mvn clean test -Dgroups=mobile -Dselenide.browser=com.tmb.pro
 
 Local-Ios | ```mvn clean test -Dgroups=mobile -Dselenide.browser=com.tmb.provider.mobile.local.IosAppProvider```
 
-Lambda-Test-Android | ```mvn clean test -Dgroups=mobile -Dselenide.browser=com.tmb.provider.mobile.remote.LambdaTestAndroidAppProvider```
+Lambda-Test-Android
+- To run your native android app test utilizing a real device on LambdaTest cloud, upload the sample android apk from [here](https://github.com/MohammedKamle/TheOneFramework/tree/main/apps) using the LambdaTest app upload [api](https://www.lambdatest.com/support/docs/upload-your-mobile-app/). This will give you `app_url` in response which we need to set for lambdaTestAppUrl in [config.properties](https://github.com/MohammedKamle/TheOneFramework/blob/main/src/test/resources/config.properties)
```

Comment:
Please change the [here] and [config.properties] to refer to amuthansakthivel github

Reply:
@amuthansakthivel  will the below work 
"To run your native android app test utilizing a real device on LambdaTest cloud, upload the sample android [apk](https://github.com/MohammedKamle/TheOneFramework/tree/main/apps) using the LambdaTest app upload [api](https://www.lambdatest.com/support/docs/upload-your-mobile-app/). This will give you `app_url` in response which we need to set for [lambdaTestAppUrl](https://github.com/MohammedKamle/TheOneFramework/blob/main/src/test/resources/config.properties)"

Reply:
@amuthansakthivel got it, done the changes 👍